### DISCUSSION TITLE
PNP Filter Checkbox IOS issue

### DISCUSSION THREAD

**Discussion by SPUser0423** [microsoft-search/pnp-modern-search#3739] [General]
Hello, 
We are using PNP filter web part to display departments and enable multiple selections via checkboxes. It works smoothly on Windows browsers and Android but encounters touch and hover problems on iPhones.

When deselecting a checkbox value in Iphone /Ipad, the visual state doesn't update immediately on iPhones/Ipad browsers. However, tapping elsewhere on an iPad removes the selection.

Seeking guidance on how to address this touch and hover issue to ensure immediate visual feedback when deselecting a checkbox value on iOS devices.

<img width="160" alt="IOS Screenshot" src="https://github.com/microsoft-search/pnp-modern-search/assets/169338197/1b5ec075-f7d6-490e-9cd8-1cae54025451">
This is the screenshot from IPAD, first user selected 2 values and later unselected one.
But checkbox still not removed

**Comment by wobba** [comment]
A developer needs to look at the issue and preferably do a PR with a fix :)
